### PR TITLE
Add glod-cli version 1.0.3 - The youtube-dl that written in Go

### PR DESCRIPTION
**Pull request by tieubao**
### All Submissions:

- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew/pulls) for the same update/change?

### New Formulae Submissions:

- [ ] Does your submission pass
`brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Have you built your formula locally prior to submission with `brew install <formula>`?

### CODE REVIEW

File: Library/Formula/glod-cli.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    mkdir_p buildpath/"src/github.com/dwarvesf/"
+    ln_s buildpath, buildpath/"src/github.com/dwarvesf/glod-cli"
+    Language::Go.stage_deps resources, buildpath/"src"
+
+    system "godep", "go", "build", "-o", "glod-cli", "."
+    bin.install "glod-cli"
+  end
+
+  test do
+    system bin/"glod-cli", "help"
```
Comment (mikemcquaid) [inline Library/Formula/glod-cli.rb]: Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

Comment (tieubao) [inline Library/Formula/glod-cli.rb]: Thanks @mikemcquaid, I will update it

Comment (DomT4) [inline Library/Formula/glod-cli.rb]: :bell: ?

Comment (tieubao) [inline Library/Formula/glod-cli.rb]: Hi @DomT4 , I'm a little bit busy with the deadline. This weekend will be good time for new commit

Comment (DomT4) [inline Library/Formula/glod-cli.rb]: No worries, thanks for the update!

**Comment by tieubao** [comment]
@mikemcquaid I did some changes for the test. Please help to check it. Thanks :+1:

File: Library/Formula/glod-cli.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    ln_s buildpath, buildpath/"src/github.com/dwarvesf/glod-cli"
+    Language::Go.stage_deps resources, buildpath/"src"
+
+    system "godep", "go", "build", "-o", "glod-cli", "."
+    bin.install "glod-cli"
+  end
+
+  test do
+    output = shell_output(bin/"glod-cli --version")
+    assert_match "glod-cli version #{version}\n", output
```
Comment (mikemcquaid) [inline Library/Formula/glod-cli.rb]: Can remove this test in favour of the other below.

**Comment by mikemcquaid** [comment]
Test is failing: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/43156/version=el_capitan/testReport/junit/brew-test-bot/el_capitan/test_glod_cli/

Maybe needs a `ENV["TERM"] = "xterm"`

**Comment by DomT4** [comment]
Slightly different failure now:

```
==> /usr/local/Cellar/glod-cli/1.0.3.2/bin/glod-cli http://mp3.zing.vn/bai-hat/Hello-Vietnam-Pham-Quynh-Anh/ZWZ9C8EB.html
Retrieving metadata ...
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x0 pc=0x76ad7]

goroutine 1 [running]:
panic(0x446d60, 0xc8200120e0)
	/usr/local/Cellar/go/1.6/libexec/src/runtime/panic.go:464 +0x3e6
github.com/dwarvesf/glod-cli/cmd.Action(0xc820116000)
	/private/tmp/glod-cli20160313-1449-n9x2yc/glod-cli-1.0.3.2/src/github.com/dwarvesf/glod-cli/cmd/commands.go:120 +0x1b37
github.com/codegangsta/cli.(*App).Run(0xc8200cc300, 0xc82000a580, 0x2, 0x2, 0x0, 0x0)
	/private/tmp/glod-cli20160313-1449-n9x2yc/glod-cli-1.0.3.2/Godeps/_workspace/src/github.com/codegangsta/cli/app.go:164 +0xc86
main.main()
	/private/tmp/glod-cli20160313-1449-n9x2yc/glod-cli-1.0.3.2/glod-cli.go:18 +0xe4
Error: glod-cli: failed
Failed executing: /usr/local/Cellar/glod-cli/1.0.3.2/bin/glod-cli http://mp3.zing.vn/bai-hat/Hello-Vietnam-Pham-Quynh-Anh/ZWZ9C8EB.html
```

**Comment by tieubao** [comment]
@DomT4 It's really weird that the test run successful in our laptops. The test will need to create a file and check if the file exists or not. Any problems with the runner?

**Comment by mikemcquaid** [comment]
@tieubao Maybe try downloading a YouTube video instead?

**Comment by mikemcquaid** [comment]
That failure looks like it's a bug in the software. Given this is so new and there's a bug like that: passing on this until it's more than 30 days old. Sorry!

**Pull request closed without merging**